### PR TITLE
Clarified and fixed instructions

### PR DESCRIPTION
**Pull request by davidkartuzinski**
#### Description of what you did:

Clarified and fixed instructions regarding plugin and admin folders.

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [x] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md [hunk 1]
```diff
@@ -202,7 +202,9 @@ Let's start by creating a new folder called `./extensions`. This folder needs to
 
 ### Migrating non customized plugin
 
-If you installed a plugin but never modified any files inside `./plugins/pluginName/**/*`, you can remove the `./plugins/pluginName` folder.
+If you installed a plugin but never modified any files inside `./plugins/pluginName/**/*`, you can remove the `./plugins/pluginName` folder. You may also remove the default installed plugins. This may mean that there are no plugins inside the `./plugins` folder, so you can delete the `./plugins` folder. 
+
+**Note:** If you have created a **custom plugin** leave the plugin in the `./plugin` folder. Newly created **custom plugins** are placed in the `./plugin` folder.
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md]: in the plugins folder with an s

File: docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md [hunk 2]
```diff
@@ -644,27 +646,39 @@ module.exports = {
 
 ## Migrating `admin`
 
-Customizing the admin is as simple as creating a file in the `./admin` folder of your app. You need to make sure the file you want to customize is at the same location in your `./admin` folder as it is in the `strapi-admin` package. For a reference you can look at the [source code](https://github.com/strapi/strapi/tree/master/packages/strapi-admin/admin).
+For the beta there are quite a lot of changes made to the admin:
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md]: It feels strange

File: docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md [hunk 3]
```diff
@@ -644,27 +646,39 @@ module.exports = {
...
+
+- If you have not customized anything in `./admin` folder, then you simply delete the `./admin` folder and it's contents. 
+- If you have customized any part of the `./admin` folder, then you need to keep only those modified files, locate their new location in the directory structure ([source code](https://github.com/strapi/strapi/tree/master/packages/strapi-admin/admin)), and then move it to the new location.
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md]: conjugaison issues teir locition / move it ?

File: docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md [hunk 4]
```diff
@@ -644,27 +646,39 @@ module.exports = {
...
 
 You can modify the logo of the app by creating a file `./admin/src/assets/images/logo-strapi.png` to replace the admin logo.
 
 You can do the same with any file found in the [source code](https://github.com/strapi/strapi/tree/master/packages/strapi-admin/admin) of the admin panel
 
+## Running your migrated project
+
+To run your migrated project you will now need to run `strapi develop` or `npm run develop` to run the project in watch mode (e.g auto reloading on content-type creation).
+
+To run strapi without watch mode then run `strapi start` or `npm run start`.
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md]: Add `npm run build`  before start plz

File: docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md (hunk 3 above)
Comment (derrickmehaffy) [reply docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md]: Is this french or did you just bang you head on the keyboard? 😆

File: docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md (hunk 1 above)
Comment (davidkartuzinski) [reply docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md]: FIxed

File: docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md (hunk 2 above)
Comment (davidkartuzinski) [reply docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md]: Agreed and edited.

File: docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md (hunk 3 above)
Comment (davidkartuzinski) [reply docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md]: This is corrected. @alexandrebodin .
@derrickmehaffy - franglais is a language...

File: docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md (hunk 4 above)
Comment (davidkartuzinski) [reply docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md]: handled - please review.

**Pull request merged**